src/main/java/Duke.java

```diff
@@ -1,10 +1,113 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static String logo = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+
+    public static String instructions = "Hello! Welcome to Duke. I am your personal task tracker.\n"
+            + "As of now, I can help you track Todos, Deadlines and Events. "
+            + "Mark your tasks with either \"todo\", \"deadline\" or \"event\" at the start. \n"
+            + "For deadlines and events, after your task, please enter either \"by (your deadline)\" "
+            + "or \"at (your event timing)\".\n"
+            + "To see all your tasks, enter \"list\". \n"
+            + "To mark a task as done, enter \"done (task number)\". \n"
+            + "To exit this program, enter \"bye\". \n"
+            + "And that's all! Hope you find me helpful! :) \n";;
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String line;
+        Scanner in = new Scanner(System.in);
+
+        Task[] taskList = new Task[100];
+        int itemCount = 0;
+
+        String greeting = "____________________________________________________________\n"
+                + "Hello! I'm Duke\n"
+                + logo
+                + "What can I do for you?\n"
+                + "Enter \"help\" if you need help using me! \n"
+                + "____________________________________________________________\n" ;
+
+        String goodbye = "____________________________________________________________\n"
+                + "Bye. Hope to see you again soon!\n"
+                +  "____________________________________________________________\n";
+
+        System.out.println(greeting);
+
+        while(true) {
+            line = in.nextLine();
+
+            boolean isTodoTask = line.startsWith("todo ");
+            boolean isDeadlineTask = line.startsWith("deadline ");
+            boolean isEventTask = line.startsWith("event ");
+            boolean isProperTask = isTodoTask || isDeadlineTask || isEventTask;
```

Comment:
Perhaps see if you can shorten this whole while loop by extracting some of the code?